CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ details for the channel changes can be observed in the console log if "debug" le
 in ESP-IDF Kconfig options.
 - Default size of ESP32 RTC slow memory from 4086 to 4096, except on ESP32-H2 where it's 3072
 - Update `byte_size/1` and `bit_size/1` to implement OTP27 match context reuse optimization OTP-18987.
+- Install AtomVM libraries source code and binaries for better dialyzer integration
```

Comment:
0.6.3 was released so this line needs to be moved up after a rebase once a release target is defined.